openmmtools/tests/test_mcmc.py

```diff
@@ -365,6 +366,20 @@ def test_metropolized_moves():
                                             'move for class {}'.format(move_class.__name__))
 
 
+def test_langevin_splitting_move():
+    """Test that the langevin splitting mcmc move works with different splittings"""
+    splittings = ["V R O R V", "V R R R O R R R V", "V { R O R } V"]
```

Comment:
Might replace the `"V { R O R } V"` with `"O { V R V } O"` or something, so that all shadow-work-generating steps are actually included in the Metropolis test?